xym/xym.py

```diff
@@ -561,14 +561,18 @@ def xym(source_id, srcdir, dstdir, strict=False, strict_examples=False, debug_le
     if is_url:
         r = requests.get(source_id, headers=rqst_hdrs)
         if r.status_code == 200:
-            content = r.text.encode('utf8').splitlines(True)
+            content = r.text.encode('utf8').decode('utf8').splitlines(True)
```

Comment:
Not saying previous code is correct, but could you describe why this is more correct than:

```
            content = r.text.splitlines(True)
```

Reply:
So I was not really sure how this encode decode works but it was failing on python3 so I did couple of tests and found out that requests.get().text returns response in unicded string while the python3 returns utf8 encoded string. so there is no reason of doing what I ve done.. I have updated code to do different things for different python versions